package streampredictor;

message Pop {
  optional string unrolled_pattern = 1;
  optional int64 strength = 2;
  optional string first_component = 3;
  optional string second_component = 4;
  repeated string first_child_parents = 5;
  optional string belongs_to = 6;
  repeated string category_members = 7;

}

message PopManager {
  repeated Pop pattern_collection = 1;
}
